frontend/common/lang.js

```diff
@@ -3,7 +3,7 @@ import LanguageDetector from "../imports/i18next-browser-languagedetector.js"
 import { html } from "../imports/Preact.js"
 import _ from "../imports/lodash.js"
 
-import { english, nederlands, ellinika } from "./lang_imports.js"
+import { english, nederlands, ellinika, deutsch } from "./lang_imports.js"
```

Comment:
We could order this alphabetically?
```suggestion
import { deutsch, ellinika, english, nederlands } from "./lang_imports.js"
```

Reply:
yes! you can already do that in your PR